### PR TITLE
Implementation

### PR DESCRIPTION
**Pull request by boyd-nguyen**
- Added What is it for and Future direction sections to Ecosystem page
- Updated naming of categories and changed resources' categories accordingly
- Added tagging system and assigned tags to a few resources. I'm not sure how to assign activity tags to the datasets so I did a few, let me know if you want to change them.

### CODE REVIEW

File: Australian Digital Observatory/content/the-ado-ecosystem/contents.lr [hunk 1]
```diff
@@ -19,14 +19,59 @@ This modular ecosystem approach gives flexibility, allowing researchers to pick
...
+
+The chart below illustrates the research workflow, as well as the specific tools used in each phase.
+
+<img src="example-workflow.png" alt="DO example workflow for Twitter analysis" class="img-fluid rounded mx-auto d-block" >
+
+<br>
+
+### Multi-platform content analysis
+
+Many research projects require data from multiple sources. For example, a researcher is interested in the dynamics and development of content and networks related to the Critical Race Theory in light of the Black Lives Matter movement. Analyses would require data from Twitter, Youtube, Reddit, Wikipedia, as well as scholarly citation networks. A set of different bespoke tools are developed and used to support this workflow:
```
Comment (Quiet27) [inline Australian Digital Observatory/content/the-ado-ecosystem/contents.lr:57]:
```suggestion
Many research projects require data from multiple sources. For example, a researcher is interested in the dynamics and development of content and networks related to the Critical Race Theory in light of the Black Lives Matter movement. Analyses would require data from Twitter, YouTube, Reddit, Wikipedia, as well as scholarly citation networks. A set of different bespoke tools are developed and used to support this workflow:
```

**Review by Quiet27 — CHANGES_REQUESTED** [review]
Looks great! Minor changes suggested (change Youtube to YouTube), and I made a couple of other commits to fix some typos in some of the resources which I hadn't noticed until now.

**Comment by betsybookwyrm** [comment]
Rebased this branch on main and resolved merge conflicts.

**Comment by betsybookwyrm** [comment]
So what I've been trying to figure out around the activity tags is whether we should:

1. provide a bit more context - e.g. instead of just displaying the activity tags next to the category as potentially random verbs, could we do something like:
    a. display them in a separate field, labelled something like "Data activity" or maybe there's something that makes more sense than that
    b. label/word the tags with "activity", e.g. "activity: analyse", depending on what punctuation is allowed
2. keep it a consistent taxonomy, such as by making the activity tag field a multiple select field rather than a multiple strings field - this has the benefit of being much easier to label resources as you just need to tick boxes.

However, doing either of these then pretty much loses the free-tagging ability that we have right now, where we can tag resources with other non-activity tags such as "twitter" or "network-analysis".  We hadn't planned this, but it might be nice. Would the ideal be to have two separate fields, one for the activity verb which implements 2 and maybe 1 above, and another free tagging field that works like the current tags field? 

I did a bit of looking in to the [lektor-tags plugin](https://github.com/lektor/lektor-tags) and it does not support having multiple tag fields, so we would have to implement them separately, which is fine, but I was about to start trying that out and then I realised I needed to stop and step back and talk to you all about what we actually want on the site :laughing: 

What are your thoughts? We can have a bit of a design discussion if you'd like?

**Review by Quiet27 — APPROVED** [review]

**Pull request merged**